Fix lauguage english

AzaleeX opened the pull request:
### English

Hello AzaleeX for the translation of the PowerNukkitX documentation.

I've done all the **untranslated pages in English**, then after you've published this code on `the branch master I'll tackle the api file in the US that hasn't been started yet` !

and if i can i would like to join the organization if it is possible ?

### Chinese

你好AzaleeX，感谢你对PowerNukkitX文档的翻译。

我已经完成了所有未翻译的英文页面，然后在你将这段代码发布到分支主站后，我将解决还未开始的美国的api文件!

如果可以的话我想加入这个组织?

Commit:
Update behavior_group.md

Commit:
Update memory.md

Lang English

Commit:
Update motion_controller.md

Lang English

Commit:
Update navigator.md

Lang English

Commit:
Update sensor.md

Lang English

Commit:
Update work_cycle.md

Lang English

Commit:
Update basic_knowledge.md

Lang English

Commit:
Update dev_faq.md

Lang English

Commit:
Update event_listener.md

Lang English

Commit:
Update hello_world.md

Lang English

Commit:
Update setup_dev_env.md

Lang English

Commit:
Update setup_llse_dev_env.md

Lang English

AzaleeX commented:
@minimouse0 @lt-name @SuperIceCN @CoolLoong

SuperIceCN commented on doc/en-us/dev/java/entity-ai/behavior_group.md:
```diff
@@ -1,39 +1,39 @@
+### 1.0.2 Singularity
 
-对于每个生物，在其生命周期中只存在一个行为组实例
+For each organism, there is only one instance of the behavior group in its life cycle
```
I think creature is better than organism here

AzaleeX replied:
fix on this commit d476ae851733db3d3eca74a962c507167620b074

SuperIceCN commented on doc/en-us/dev/java/entity-ai/behavior_group.md:
```diff
@@ -1,39 +1,39 @@
-- 控制器
+- Behavior, Core Behavior
+- Memory Storage
+- Sensors
```
It would be better to unify the singular and plural forms here

AzaleeX replied:
I don't see what I need to change in all of this.

SuperIceCN replied:
Maybe remove the `s` in Sensors?

AzaleeX replied:
fix on this commit d476ae851733db3d3eca74a962c507167620b074

SuperIceCN commented on doc/en-us/dev/js/dev_faq.md:
```diff
@@ -18,17 +18,17 @@ PowerNukkitX.listenEvent("cn.nukkit.event.entity.EntityDamageByEntityEvent", Eve
-PNXJS的每个JS插件都是一个独立的`Context`，这意味着，如果你使用`import`导入其他插件的模块时，会完全重新载入一次被导入的插件的模块。  
-例如以下基于[LLSE-Lib](https://github.com/PowerNukkitX/LLSE-Lib)库编写的示例:  
+## 3. Some considerations when importing modules from other JS plugins
+Each JS plugin of PNXJS is a separate `Context`, which means that if you import a module of another plugin using `import`, it will completely reload the module of the plugin being imported once.  
```
The Chinese here has context ellipsis, but if they are not supplemented when translated into English, it will lead to incorrect expression of meaning.
The complete sentence goes like this: PNXJS的每个JS插件都是处于一个独立的`Context`之中的
So if we translate like this it may be better: Each PNX-JS plugin runs in a separate `Context`

AzaleeX replied:
fix in next commit

AzaleeX replied:
fix in this commit c726061604c73240c6e1dbc554a7eeb6fcfc7182

SuperIceCN commented on doc/en-us/dev/js/dev_faq.md:
```diff
@@ -43,12 +43,11 @@ BPlugin.js
+The `onlyOnceExecute` from the [LLSE-Lib](https://github.com/PowerNukkitX/LLSE-Lib) library is designed to solve this problem.
+The first parameter of `onlyOnceExecute` passes a callback function and the second parameter passes a UUID as a unique identifier, and this callback function will only be executed once, no matter how many plugins import the plugin.
 
+But since each JS plug-in of PNXJS is a separate `Context`, the B plug-in imports the A variable of the `APlugin` plug-in, which is not initialized, while the `Context` of the B plug-in loads the A plug-in completely once, but since `onlyOnceExecute` will only be executed once globally, the A variable in the B plugin's `Context` will not be initialized
```
Please replace all `plug-in` with `plugin`.

AzaleeX replied:
fix in next commit

AzaleeX replied:
commit c726061604c73240c6e1dbc554a7eeb6fcfc7182

SuperIceCN commented on doc/en-us/dev/js/event_listener.md:
```diff
@@ -1,54 +1,54 @@
+An event object is generated whenever a specific action occurs in the server. An event is a description of some behavior or action, such as a player logging in, a
+player leaves, player sends a message, and so on. Each kind of behavior or action has an event object as its abstraction, and the event object carries some key information about the behavior or action.
+The event object contains some key information about the behavior or action, e.g., the player object that sent the event, the message string sent, etc., and the player object contains the player's
+The player object contains the player's location, life value, item bar, etc.
```
Here seems to be a repetition of the content here

AzaleeX replied:
fix next commit

AzaleeX replied:
11790750ed1420b783524860fab09c4dcbf3704d

SuperIceCN commented on doc/en-us/dev/js/event_listener.md:
```diff
@@ -1,54 +1,54 @@
 
-PNX中提供了非常多事件，您可以在 [事件对照表](../../res/事件对照表.html) 中查看。表中每个事件都标注了类名，
-您可以在插件中使用这个类名来监听这个事件，点击类名也可以进入对应的JavaDoc查看更多细节。  
+PNX provides a very large number of events that you can use in the [Event Comparison Table](../../res/事件对照表.html) in the table. Each event in the table is labeled with the name of the category of the event.
```
It should be `[Event List](../res/event_list.html)`

AzaleeX replied:
fix next commit

AzaleeX replied:
fix in commit 11790750ed1420b783524860fab09c4dcbf3704d

SuperIceCN commented:
@AzaleeX Yeah of course you're welcomed to join us.   

It would be even better if you could make some adjustments to correct the inappropriate or inconsistent areas after the translation is completed.

Commit:
Update behavior_group.md

change of the following words with before after

Sensors => Sensor
organism => creature

Commit:
Update dev_faq.md

change of the following words with before after

JS plugin of PNXJS is => PNX-JS plugin runs in

plug-in imported the `APlugin` plug-in A variable, this will be in the B plug-in `Context` completely loaded once A plug-in => plugin imported the `APlugin` plugin A variable, this will be in the B plugin `Context` completely loaded once A plugin

Commit:
Update event_listener.md

change of the following words with before after

1. rephrasing line 7 to 10
2. redirect change to ../res/event_list.html

AzaleeX commented:
@SuperIceCN I changed everything, all the little things that you find wrong in my English translation, everything is fixed in the 3 commits I just made:

1. d476ae851733db3d3eca74a962c507167620b074 (behavior_group.md)
2. c726061604c73240c6e1dbc554a7eeb6fcfc7182 (dev_faq.md)
3. 11790750ed1420b783524860fab09c4dcbf3704d (event_listener.md)

SuperIceCN reviewed (approved):
LGTM~

AzaleeX commented:
Thank you for my request to join the organization, what exactly do I have to do to be taken ?
😀

AzaleeX commented:
@SuperIceCN Thank you very much I would like to know if on the discord there is a particular grade for us contributors ? if yes I would like to have it my discord ID: azalee.x

Tanks You 😊
